graph/services/userEvents/userEvents.service.go

```diff
@@ -44,8 +45,7 @@ func AddMembersToEvent(event string, body model.AddMemberInput) (int, error) {
 		}
```

Comment:
You could have used goqu query builder to insert multiple rows using `goqu.Record`

https://doug-martin.github.io/goqu/docs/inserting.html#insert-record